_sass/jekyll/layout.scss

```diff
@@ -251,3 +251,7 @@
 //         }
 //     }
 // }
+
+.no-scroll {
+  overflow-y: hidden;
+}
```

Comment:
this class was made to stop grid-block from scrolling the sidebar at mobile sizes, looking into a better solution because this gets overwritten.